### PR TITLE
PROGRAMMES-6554 Removing references to iPlayer Radio

### PR DESCRIPTION
**Pull request by votemike**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6554

Can anyone find any references I've missed?

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: translations/programmes/en.po [hunk 1]
```diff
@@ -1538,11 +1538,11 @@ msgstr "Sorry there are no profiles for %1"
 msgid "programme_availability_none"
 msgstr "This programme is not currently available on BBC iPlayer"
 
-#. English: "This programme is not currently available on BBC iPlayer Radio".
+#. English: "This programme is no longer available".
 #. Example URL: http://www.bbc.co.uk/programmes/b03kpkyz .
 #. Priority for: Programmes
 msgid "programme_availability_none_radio"
-msgstr "This programme is not currently available on BBC iPlayer Radio"
+msgstr "This programme is no longer available"
```
[the remark this answers is not in the document]
Comment (votemike) [reply translations/programmes/en.po]: Shall I change it to "This programme is not currently available"?

**Comment by votemike** [comment]
Not sure what I was thinking....
I think I couldn't find references to those strings in the codebase, so didn't change them.

File: translations/programmes/en.po (hunk 1 above)
[the remark this answers is not in the document]
Comment (BjornTwachtmann) [reply translations/programmes/en.po]: I think so, but check with Claudio

File: translations/programmes/en.po [hunk 2]
```diff
@@ -570,7 +570,7 @@ msgstr "Sorry, this episode is not currently available on BBC iPlayer Radio"
 msgid "episode_availability_past"
 msgstr "Sorry, this episode is no longer available"
 
-#. English: "Sorry, this episode is no longer available on BBC iPlayer Radio".
+#. English: "Sorry, this episode is no longer available".
 #. Priority for: Programmes
 msgid "episode_availability_past_radio"
 msgstr "Sorry, this episode is no longer available on BBC iPlayer Radio"
```
Comment (BjornTwachtmann) [inline translations/programmes/en.po]: You need to change the msgstr, not just the comments! There's a few of these

Comment (BjornTwachtmann) [reply translations/programmes/en.po]: or delete it if that's what the ticket says, but the comments are out of sync with the text now

File: src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/PlayoutPresenter.php [hunk 3]
```diff
@@ -137,10 +137,13 @@ public function getNotAvailableTranslation(): string
...
 
+        if (!$this->episode->isTleo()) {
+            return 'episode_availability_none';
+        }
+
         $suffix = $this->episode->isRadio() ? '_radio' : '';
-        $prefix = $this->episode->isTleo() ? 'programme_' : 'episode_';
 
-        return $prefix . 'availability_none' . $suffix;
+        return 'programme_availability_none' . $suffix;
```
Comment (votemike) [inline src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/PlayoutPresenter.php]: @BjornTwachtmann Now that I've updated more of the language strings, should I revert the changes to this file?

File: src/DsAmen/Presenters/Section/Map/SubPresenter/on_demand.html.twig [hunk 4]
```diff
@@ -45,7 +45,7 @@
                 <a
                     data-linktrack="{{ presenter.getAllLinkLocation() }}"
                     {% if presenter.getProgrammeContainer().isRadio() %}
-                        title="{{ tr('all_episodes_iplayer_radio_title', {'%1': presenter.getProgrammeContainer().getTitle()}) }}"
+                        title="{{ tr('see_all_available_from_radio', {'%1': presenter.getProgrammeContainer().getTitle()}) }}"
```
Comment (votemike) [inline src/DsAmen/Presenters/Section/Map/SubPresenter/on_demand.html.twig]: @BjornTwachtmann Now that I've updated more of the language strings, should I revert the changes to this file?

**Comment by Tasiobg** [comment]
I think `SchemaHelper` and the test should be updated as well, there is some "iPlayer Radio" occurrences.

What are we planing to do with "iPlayer Radio" occurrences in translations that are not used yet? I think we could remove them now when we know there is no going to be "iPlayer Radio" any more.

File: src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/PlayoutPresenter.php (hunk 3 above)
Comment (BjornTwachtmann) [reply src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/PlayoutPresenter.php]: revert this if it works without

File: src/DsAmen/Presenters/Section/Map/SubPresenter/on_demand.html.twig (hunk 4 above)
Comment (BjornTwachtmann) [reply src/DsAmen/Presenters/Section/Map/SubPresenter/on_demand.html.twig]: yeah, revert it

**Comment by votemike** [comment]
@BjornTwachtmann I think this should be ready for you to review again.
